cypress/e2e/default/dynamic-routes.cy.ts

```diff
@@ -66,8 +66,7 @@ describe('Dynamic Routing', () => {
       expect(res.status).to.eq(200)
       // expect 'odb' until https://github.com/netlify/pillar-runtime/issues/438 is fixed
       expect(res.headers).to.have.property('x-nf-render-mode', 'odb')
-      // expect 'Bitten' until the above is fixed and we can test for fallback 'Loading...' message
-      expect(res.body).to.contain('Bitten')
+      expect(res.body).to.contain('Loading...')
```

Comment:
This works now @orinokai. It was failing previously not finding `Bitten` which was expected pre the changes in this PR.